_layouts/base.html

```diff
@@ -10,6 +10,40 @@
         {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
         <link href="{{ site.baseurl }}/css/custom.css" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
+        <!--Below is the schema annotation for blogposts-->
+        {% if page.layout == 'post' %}
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org",
+            "@type": "BlogPosting",
+            "mainEntityOfPage": {
+              "@type": "WebPage",
+              "@id": "{{ site.url }}/posts/"
+              },
+            "headline": "{{ page.title }}",
+            "description": "{{ page.excerpt | strip_html | strip_newlines | truncate: 200 }}",
+            "image": "{{ page.image }}",
+            "author": [
+            {%- for author in page.authors -%}
+              {
+              "@type": "Person",
+              "name": "{{ site.data.people[author].first-name }} {{ site.data.people[author].last-name }}",
+              "url": "{{ site.url }}/people#{{ author }}",
+              },
```

Comment:
The comma for the last iteration might cause an error. Did you try to validate with the authors?

Reply:
It displayed all the authors...separated by commas. I didn't catch any errors.

Reply:
I got an error when I used Schema.org validator: https://validator.schema.org/#url=https%3A%2F%2Fdeploy-preview-605--ols-bebatut.netlify.app%2Fposts%2F2023%2F03%2F21%2Fols-7-announcement%2F

Reply:
Ah, yes. I see it now. Wow...